build-ioc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -ex
+
+cp /opt/epics/RELEASE configure/
+cat << EOF > configure/CONFIG_SITE.local
+STATIC_BUILD=YES
+FULL_STATIC_BUILD=YES
+USR_LDFLAGS = -static-pie
+EOF
+
+git config --global --add safe.directory $PWD
+
+make distclean
+make
+make clean
+make -C iocBoot
+
+version=$(git describe)
+
+echo $version > rffe-epics-ioc-version
+echo ".git/" > /tmp/globs
+# this trick is done so the resulting tarball has a root rffe-epics-ioc/ directory
+(cd .. && tar caf rffe-epics-ioc/rffe-epics-ioc-${version}.tar.gz -X /tmp/globs rffe-epics-ioc/)
```

Comment:
Why do we need this trick? To avoid having to create a directory before extracting the tarball in the deploy machine?

Reply:
I've been bothered by this in the past, but I think it makes sense after all.